tensorflow/compiler/tests/plugin_config.tpl

```diff
@@ -0,0 +1,14 @@
+# Configuration for a plugin
```

Comment:
(we may need copyright notices on all files, so if you can add one, it would be good!)